packages/core/theme-module/theme/pages/Category.vue

```diff
@@ -254,24 +267,47 @@ import {
   getProductVariants
 } from '<%= options.helpers %>';
 
+const defaultPagination = {
+  page: 1,
+  itemsPerPage: 20
+};
+
+const perPageOptions = [20, 40, 100];
```

Comment:
Should these numbers be configurable? Or it's fine for now?

Reply:
Imho default pagination options should be taken from API client config and then be configurable either globally from it or per composable

Reply:
To be resolved in #296